admin/btop/Makefile

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Why GPL-3.0? It's your choice but whole openwrt is gpl-2.0-only? ^^

Reply:
It's a long story... I will remove this header later today ;)

Reply:
You can leave it GPL-3.0. I don't care. ;)
I was just interested.